src/TimePeriodPicker.js

```diff
@@ -16,7 +16,8 @@ const TimePeriodPicker = (params) => {
   };
   console.log(params.startDate);
   console.log(params.endDate);
-
+  console.log(startingDate);
+  console.log(endingDate);
```

Comment:
are these needed?

Reply:
I had added them to remove an unused variable error. I'll disable lint instead.